blog.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Adopt a Dog Blog</title>
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  </head>
+  <body>
+    <header class="header">
+      <img class="header-img" src="images/adoptadog-logo.svg"  alt="Logo">
+      <h1 class="header-text">Seattle's premier dog adoption agency</h1>
+    </header>
+  <section class="navbar">
+    <nav>
+      <div class="nav">
```

Comment:
This is an unnecessary div. You already have a wrapping container with the nav element